### PR TITLE
new Subtitles for ZAP Chat Video 17

### PR DESCRIPTION
**Pull request by 8linkz**
Heyho - here are the new subtitles for the Youtube video.

Simon confused me a bit with “open API” and "open AI" - please take an additional look here. 

As a reminder: empty lines are created when whisper halucinates here. I then delete the text, but the timestamp MUST remain, otherwise the import into YouTube will not work.

Video: https://www.youtube.com/watch?v=hZ9yeXK2DLY

### CODE REVIEW

**Comment by 8linkz** [comment]
git is driving me crazy - no idea how I resolved it - but it looks better now than before xD

Well, before I break anything else here and tinker around, I better leave it like this...

File: subtitles/ZAP Chat/ZAP_Chat_17_Fuzz_AI_Files.srt [hunk 1]
```diff
@@ -0,0 +1,828 @@
...
+00:05:47,400 --> 00:05:50,400
+So let's have a look at some of the responses.
+
+95
+00:05:50,400 --> 00:05:52,400
+So here you go.
+
+96
+00:05:52,400 --> 00:06:01,400
+You can actually see I'm based on GPT-4, which is interation of open APIs, generative pre-trained transform models, etc., etc.
```
Comment (kingthorin) [inline subtitles/ZAP Chat/ZAP_Chat_17_Fuzz_AI_Files.srt]: interaction ?

File: subtitles/ZAP Chat/ZAP_Chat_17_Fuzz_AI_Files.srt [hunk 2]
```diff
@@ -0,0 +1,828 @@
...
+00:10:41,400 --> 00:10:42,400
+Shame.
+
+180
+00:10:42,400 --> 00:10:43,400
+
+
+181
+00:10:43,400 --> 00:10:46,400
+So a well-doiled response back, right?
```
Comment (kingthorin) [inline subtitles/ZAP Chat/ZAP_Chat_17_Fuzz_AI_Files.srt]: oiled ?

File: subtitles/ZAP Chat/ZAP_Chat_17_Fuzz_AI_Files.srt (hunk 1 above)
Comment (8linkz) [reply subtitles/ZAP Chat/ZAP_Chat_17_Fuzz_AI_Files.srt]: done

File: subtitles/ZAP Chat/ZAP_Chat_17_Fuzz_AI_Files.srt (hunk 2 above)
Comment (8linkz) [reply subtitles/ZAP Chat/ZAP_Chat_17_Fuzz_AI_Files.srt]: done

**Review by psiinon — APPROVED** [review]

**Review by kingthorin — APPROVED** [review]

**Pull request merged**